Rewrite cluster version instructions as Markdown (#44)
Data-Linkage/Clerical_Resolution_Online_Widget: version1_tkinter/README.md

Reviewer: ```suggestion
[Cluster_Version_Instructions]: ./docs/Cluster_Version_Instructions.md
[crow-contributing]: ../CONTRIBUTING.md
```

Adding link to cluster version instructions used in a suggestion above.

Reviewer: ```suggestion
[Instructions_Clerical_Coordinators_pairwise]: ./docs/Instructions_Clerical_Coordinators_pairwise.pdf
[python-docs-venv]: https://docs.python.org/3/library/venv.html
```

Adding link to pairwise instructions used in a suggestion above.

- Python 3.9 or higher
- `tkinter` (included with standard Python installations)
- Additional dependencies listed in `requirements.txt`
- The data to be matched in CSV format. Help with preparing the pairwise CSV can
  be found in

>   [the pairwise instructions][Instructions_Clerical_Coordinators_pairwise]. Help

Reviewer: ```suggestion
  [the pairwise instructions][clerical-coordinator-pairwise-instructions]. Help
```

Change link reference for consistency.

  with preparing the clusters CSV can be found in

>   [the cluster instructions][Cluster_Version_Instructions].

Reviewer: ```suggestion
- The data to be matched in CSV format. Help with preparing the pairwise CSV can
  be found in [the pairwise
  instructions][clerical-coordinator-pairwise-instructions]. Help with preparing
  the clusters CSV can be found in [the cluster
  instructions][cluster-instructions].
```

Edited the link references for consistency.

- Additional dependencies listed in `requirements.txt`
- **Optional**: An [IDE][ide] (e.g. Spyder, Visual Studio (VS) Code, PyCharm
  etc.)

> - The data to be matched in csv format. Help with preparing the pairwise csv can be found in part 7 of: version1_tkinter/docs/Instructions_Clerical_Coordinators_pairwise. Help with preparing the clusters csv can be found in: version1_tkinter/docs/Cluster_Version_Instructions

Reviewer: ```suggestion
- Additional dependencies listed in `requirements.txt`
- The data to be matched in CSV format. Help with preparing the pairwise CSV can
  be found in
  [the pairwise instructions][Instructions_Clerical_Coordinators_pairwise]. Help
  with preparing the clusters CSV can be found in
  [the cluster instructions][Cluster_Version_Instructions].
- **Optional**: An [IDE][ide] (e.g. Spyder, Visual Studio (VS) Code, PyCharm
  etc.)
```

- Wrapped lines to 80 characters.
- Within bodies of text we can use [links](https://www.markdownguide.org/basic-syntax/#links) and pop the URLs/paths at the end of the document (see the suggestion at the end of the doc).

versions of CROW1. Your choice will depend on the format of the data that you
will be clerically reviewing.

CROW_pairwise should be used when making a simple match or non-match decision between 2 records.

> CROW_clusters should be used when you have a group of records (these can be from a single or multiple data sources). This is useful when resolving conflicting links.

Reviewer: ```suggestion
CROW_pairwise should be used when making a simple match or non-match decision
between 2 records.

CROW_clusters should be used when you have a group of records (these can be from
a single or multiple data sources). This is useful when resolving conflicting
links.
```

Just wrapping lines to 80 characters.

[contributing guidelines][crow-contributing], or [contact us directly via
email][email].

> [Cluster_Version_Instructions]: ./docs/Cluster_Version_Instructions.md

Reviewer: ```suggestion
[clerical-coordinator-pairwise-instructions]: ./docs/Instructions_Clerical_Coordinators_pairwise.pdf
[cluster-instructions]: ./docs/cluster_version_instructions.md
```

Renamed link references for consistency.

[crow-contributing]: ../CONTRIBUTING.md
[crow-issues]: https://github.com/Data-Linkage/Clerical_Resolution_Online_Widget/issues
[crow-new-issue]: https://github.com/Data-Linkage/Clerical_Resolution_Online_Widget/issues/new
[email]: mailto:linkage.hub@ons.gov.uk
[ide]: https://en.wikipedia.org/wiki/Integrated_development_environment

> [Instructions_Clerical_Coordinators_pairwise]: ./docs/Instructions_Clerical_Coordinators_pairwise.pdf

Reviewer: ```suggestion
```

Link reference moved further up.
